pkg/storage/apis/obot.obot.ai/v1/run.go

```diff
@@ -23,6 +24,11 @@ const (
 	AuthProviderSyncAnnotation  = "obot.ai/auth-provider-sync"
 )
 
+var (
+	_ fields.Fields = (*Run)(nil)
+	_ DeleteRefs    = (*Run)(nil)
+)
```

Comment:
What does this do?

Reply:
It creates a compile-time check that `*Run` implements these interfaces.